src/Files/SafetyHelpers/ExceptionReporters/IStorageFolder.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Common.SafetyHelpers.ExceptionReporters
+{
+    internal interface IStorageFolder
```

Comment:
Are these placeholders for something? What for?

Reply:
Oh no! I must have accidentally selected "Create in new file" while resolving namespaces in VS 😅 That's obviously not to stay and I didn't notice it